src/main.ts

```diff
@@ -15,29 +15,20 @@ async function bootstrap() {
   });
   const csrfProtection = csurf({
     cookie: {
-      key: 'XSRF-TOKEN',// can set a custome cookie name.
-      path: "/",
+      key: 'XSRF-TOKEN', // can set a custome cookie name.
+      path: '/',
       httpOnly: true,
       secure: false,
       // maxAge: -20000, // 14 days,
       maxAge: 14 * 24 * 60 * 60 * 1000, // 14 days,
       // maxAge: 14 * 24 * 60 * 60 * 1000, // 14 days,
       sameSite: 'lax',
-      ignoreRoutes: ['admin', 'api'],
+      // ignoreRoutes: ['admin', 'api'],
```

Comment:
The cookies config doesn't have an ignored routes

All other changes here are via prettier and the formatter config you have set up